validation/superchain-genesis_test.go

```diff
@@ -14,9 +14,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func testGenesisHash(t *testing.T, chainID uint64) {
-	skipIfExcluded(t, chainID)
-
+var testGenesisHash SubTestForChain = func(t *testing.T, chain *ChainConfig) {
```

Comment:
You do this for all the test functions. Is it necessary? I think if you define
```go
type (
	SubTest         = func(t *testing.T)
	SubTestForChain = func(t *testing.T, chain *ChainConfig)
)
```
as aliases, instead of new types (by adding `=`), you could still use the normal functions, which is maybe cleaner than defining global function `var`s.

Reply:
It's not necessary, I don't think new types vs aliases makes much difference here (we can keep the normal functions if we want). My efforts here are to try and establish a convention for any future dev to follow when adding a new validation check. By declaring the function as a `SubTestForChain` they will get a hint about the function signature they need to adhere to.

Reply:
If you define the type alias, you still hint at the function signature, but can keep on defining normal functions instead of global function `var`s.